spm-monitor-generic/src/main/java/com/sematext/spm/client/postgresql/PostgreSQLVersionCheck.java

```diff
@@ -0,0 +1,82 @@
+package com.sematext.spm.client.postgresql;
```

Comment:
Please add the standard copyright header

Reply:
Done commit 7baf05e